crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -2203,6 +2208,11 @@ where
                 if self.enabled(Rule::UncapitalizedEnvironmentVariables) {
                     flake8_simplify::rules::use_capital_environment_variables(self, expr);
                 }
+                if self.enabled(Rule::UnnecessaryListAllocationForFirstElement) {
+                    ruff::rules::unnecessary_list_allocation_for_first_element(
+                        self, value, slice, *range,
+                    );
```

Comment:
Nit: We could pass the whole subscript instead of each individual field (also cheaper, because it passes a single reference instead of 3)
```suggestion
                    ruff::rules::unnecessary_list_allocation_for_first_element(
                        self, subscript
                    );
```


And on line 2142:

```rust
   ExprSubscript(subscript @ ast::ExprSubscript { ... })
```